funcx_sdk/parsl/app/errors.py

```diff
@@ -4,7 +4,6 @@
 from typing import Any, Callable, TypeVar, Union
 
 import dill
-from six import reraise
```

Comment:
Odd that 1) this is for py2 vs 3 compatibility which we don't care about and 2) we were never using this to start with.

Reply:
Given that the "fake `parsl`" is being reverted, it might not matter. But yes, I thought it was odd that this was included.